mdsuite/calculators/green_kubo_distinct_diffusion_coefficients.py

```diff
@@ -231,27 +250,36 @@ def _post_operation_processes(self, species: Union[str, tuple] = None):
                       "Analysis": self.analysis_name,
                       "Subject": list(species),
                       "data_range": self.data_range,
-                      'data': [{'x': np.mean(result), 'uncertainty': np.std(result) / (np.sqrt(len(result)))}]
+                      'data': [{'x': np.mean(result),
+                                'uncertainty': np.std(result) /
+                                               (np.sqrt(len(result)))}]
                       }
         self._update_properties_file(properties)
         # Update the plot if required
         if self.plot:
-            plt.plot(np.array(self.time) * self.experiment.units['time'], self.vacf, label=species)
+            plt.plot(np.array(self.time) * self.experiment.units['time'],
+                     self.vacf,
+                     label=species)
+            plt.vlines((np.array(self.time) * self.experiment.units['time'])[
+                           self.integration_range], min(self.vacf),
+                       max(self.vacf))
```

Comment:
I guess this will be changed to the visualization module eventually?

Reply:
Yes. In fact the idea is to have a trigger in the plotting that allows for a re-calculation of the integral by user input.